front/app/containers/Admin/projects/project/index.tsx

```diff
@@ -361,6 +360,10 @@ export class AdminProjectsProjectIndex extends PureComponent<
         });
       }
 
+      const showDropdownButton =
+        project.attributes.process_type === 'timeline' &&
+        numberIdeationPhases > 1;
+
```

Comment:
The changes in this file fix a UI bug where the "Add idea" button flickers/glitches.